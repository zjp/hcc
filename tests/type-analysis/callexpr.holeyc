int b;
bool a;
void func(int f, int b, bool c){
	int a;
	a = func(7,"hello");
}
