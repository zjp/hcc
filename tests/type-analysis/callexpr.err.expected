FATAL [5,6]: Function call with wrong number of args
FATAL [5,6]: Invalid assignment operand
Type Analysis Failed
